daprdocs/data/components/state_stores/generic.yaml

```diff
@@ -42,6 +42,17 @@
     etag: true
     ttl: false
     query: false
+- component: Coherence
```

Comment:
This is an oracle DB and hence this should be in the oracle.yaml section as opposed to generic.yaml?